plugins/package-managers/conan/src/funTest/kotlin/ConanFunTest.kt

```diff
@@ -41,6 +43,11 @@ import org.ossreviewtoolkit.utils.test.patchActualResult
  * the Conan cache before executing the tests, i.e. rm -Rf ~/.conan2/p/.
  */
 class ConanFunTest : StringSpec({
+    beforeEach {
+        val homeDirectory = System.getProperty("user.home")
+        File(homeDirectory, ".conan2/p").deleteRecursively()
```

Comment:
Should this be `.conan2/data/p` to align with [this](https://github.com/oss-review-toolkit/ort/blob/81670f83aa806570f55b44fe1da39fdd5e7830db/plugins/package-managers/conan/src/main/kotlin/ConanV2Handler.kt#L50)?

Reply:
But in general I'm feeling uncomfortable with deleting user data (that not necessarily was created by ORT) when running tests.

Can you maybe instead find a way to run Conan in a way that it does not make use of cached results when running tests?

Reply:
Maybe just temporarily set the `CONAN_USER_HOME` environment variable to an empty directory "[to fully move the cache](https://github.com/conan-io/conan/issues/2250#issuecomment-354998297)"?

Reply:
Looks like [Conan 2 uses `CONAN_HOME` instead](https://docs.conan.io/2/reference/environment.html#conan-home).

Reply:
> Should this be `.conan2/data/p` to align with [this](https://github.com/oss-review-toolkit/ort/blob/81670f83aa806570f55b44fe1da39fdd5e7830db/plugins/package-managers/conan/src/main/kotlin/ConanV2Handler.kt#L50)?

Congrats you found the bug :smile: 

ORT stashes this folder https://github.com/oss-review-toolkit/ort/blob/81670f83aa806570f55b44fe1da39fdd5e7830db/plugins/package-managers/conan/src/main/kotlin/Conan.kt#L189 so it solves the test problems.